docs/guide/faq/README.md

```diff
@@ -16,6 +16,7 @@ This problem can be divided in 2 categories; no logging is shown at all OR inter
 - If you are using a Raspberry Pi, try disconnecting any other USB devices. If after that pairing works, try connecting the USB devices via a powered USB hub.
 - Make sure that any other Zigbee networks/hubs are powered down. When you e.g. want to pair an IKEA bulb which was first paired to the IKEA gateway make sure to power down the IKEA gateway. If that doesn't help also try powering down all devices that are connected to the IKEA hub.
 - If it's a battery powered device, try replacing the battery.
+- You are running an older firmware, make sure the latest firmware is installed.
```

Comment:
The older coordinator firmware is not the actual problem, a reflash of the same firmware will probably also fix it. A few lines below:

```
  - If none of the above helps, try to reflash the coordinator (does not require repairing of already paired devices).
```